sdk/template/template/package.json

```diff
@@ -57,7 +68,7 @@
   "bugs": {
     "url": "https://github.com/Azure/azure-sdk-for-js/issues"
   },
-  "homepage": "https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/template/template/",
+  "homepage": "https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/template/template/README.md",
```

Comment:
Should we update the linter rule to enforce this (not as part of this PR)? Right now it allows both

Reply:
@maorleger I think @ramya-rao-a had a post about what to put in the homepage here on our Teams channel a while ago but IIRC we did not decide on something concrete yet. Perhaps we can bring this up in one of our weekly meetings.

Reply:
Yeah, this is still up in the air. This is what Text Analytics does, Form Recognizer does it the other way.